### ISSUE TITLE
Pipeline example

### ISSUE THREAD

**Issue by networkop** [wobcom/transceiver-exporter#3]
👋 heya, 
would it be possible to include an example of what can be achieved with this information? I don't have much experience with this but I understand that increasing power levels could be an indication of a dying transceiver. I think it would great to have an example of, say, alert that triggers off 99th percentile exceeding a certain baseline value (or something similar).

**Comment by vidister** [comment]
Good idea, I can include some of our Grafana panels and Alerts as examples.
Put this on my ToDo-List for the next weeks.
